autoscaler/migration.go

```diff
@@ -1,4 +1,4 @@
-package main
+package autosacler
```

Comment:
opened https://github.com/odigos-io/odigos/pull/2442 to fix this typo (looks like I missed it before this merged)